docs/content/misc/math-expr.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc_page
+---
+
+This expression language supports following operators (listed in increasing order of precedence).
+
+|Operators|Description|
+|---------|-----------|
+|and,or|Binary Logical AND, OR|
+|<, <=, >, >=, ==, !=|Binary Comparison|
+|+, -|Binary additive|
+|*, /, %|Binary multiplicative|
+|^|Binary power op|
+|!, -|Unary NOT and Minus|
+
+long and double data types are supported, number containing a dot is interpreted as a double or else a long. That means, always add a '.' to your number if you want it intepreted as a double value.
+You can use variables inside the expression which must start with an alphabet or '_' or '$' and can contain a digit as well in subsequent characters. For logical operators, positive number is considered a boolean true and false otherwise.
```

Comment:
It's the variables that must start with a certain character, not the expression. I think this is better:

Expressions can contain variables. Variable names may contain letters, digits, '_' and '$'.  Names must not begin with a digit.

For logical operators, a number is true if and only if it is positive.
(you're saying that 0 is false, is that right?)